Tests/SwiftParserTest/IncrementalParsingTests.swift

```diff
@@ -61,3 +115,46 @@ public class IncrementalParsingTests: XCTestCase {
     XCTAssertEqual(origStructB.parent!.id, reusedNode.parent!.id)
   }
 }
+
+// MARK: helper functions
+extension IncrementalParsingTests {
```

Comment:
Since you aren’t using any members (not that there are any) of `IncrementalParsingTests`, I would just make `getDiffOfTwoString` a top-level function. That’s also what we do with all the help functions in `_SwiftSyntaxTestSupport`.